universal_computation/experiment.py

```diff
@@ -57,6 +62,20 @@ def experiment(
         use_embeddings = False
         experiment_type = 'classification'
 
+    elif task == 'mnist-add':
```

Comment:
I believe you should also make a note which version of this task we should test. Is it a regression one?

Reply:
Moving to another issue.